src/zodbupdate/main.py

```diff
@@ -28,44 +28,51 @@
 logger = logging.getLogger('zodbupdate')
 
 
-parser = optparse.OptionParser(
+parser = argparse.ArgumentParser(
     description=("Updates all references to classes to "
                  "their canonical location."))
-parser.add_option(
+exclusive_group = parser.add_mutually_exclusive_group()
+exclusive_group.add_argument(
     "-f", "--file",
     help="load FileStorage")
-parser.add_option(
+exclusive_group.add_argument(
     "-c", "--config",
     help="load storage from config file")
-parser.add_option(
+parser.add_argument(
     "-n", "--dry-run", action="store_true",
     help="perform a trial run with no changes made")
-parser.add_option(
+parser.add_argument(
     "-s", "--save-renames",
     help="save automatically determined rename rules to file")
-parser.add_option(
+parser.add_argument(
     "-q", "--quiet", action="store_true",
     help="suppress non-error messages")
-parser.add_option(
+parser.add_argument(
     "-v", "--verbose", action="store_true",
     help="more verbose output")
-parser.add_option(
+parser.add_argument(
     "-o", "--oid",
     help="start with provided oid in hex format, ex: 0xaa1203")
-parser.add_option(
+parser.add_argument(
     "-d", "--debug", action="store_true",
     help="post mortem pdb on failure")
-parser.add_option(
+parser.add_argument(
     "--pack", action="store_true", dest="pack",
     help=("pack the storage when done. use in conjunction of -c "
           "if you have blobs storage"))
-parser.add_option(
-    "--convert-py3", action="store_true",  dest="convert_py3",
+parser.add_argument(
+    "--convert-py3", action="store_true", dest="convert_py3",
     help="convert pickle format to protocol 3 and adjust bytes")
-parser.add_option(
+parser.add_argument(
     '--encoding', dest="encoding",
     help="used for decoding pickled strings in py3"
 )
+parser.add_argument(
+    '--encoding-fallback', dest="encoding_fallbacks", nargs="*",
+    help="Older databases may have other encoding stored than 'utf-8', like latin1."
+         "If an encoding error occurs, fallback to the given encodings "
+         "and issue a warning.",
```

Comment:
Is it worth to mention here that this argument can be used multiple times?

Reply:
The `argparse` help output shows it anyway.